src/components/Pagination/PaginationRow.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import PaginationRowAmountOfPages from './PaginationRowAmountOfPages';
+import PaginationRowButtonGroup from './PaginationRowButtonGroup';
+import PaginationRowItems from './PaginationRowItems';
+import PaginationRowBack from './PaginationRowBack';
+import PaginationRowForward from './PaginationRowForward';
+import { PAGINATION_VIEW_TYPES, PAGINATION_VIEW } from './constants';
+import { Form, FormControl, FormGroup, ControlLabel } from '../Form';
+import { DropdownButton } from '../Button';
+import { MenuItem } from '../MenuItem';
+
+/**
+ * PaginationRow component for Patternfly React
+ */
+const PaginationRow = ({
+  className,
+  contentView,
+  viewType,
+  pagination,
+  amountOfPages,
+  itemCount,
+  itemsStart,
+  itemsEnd,
+  messages,
+  dropdownButtonId,
+  onPerPageSelect,
+  onFirstPage,
+  onPreviousPage,
+  onPageInput,
+  onNextPage,
+  onLastPage
+}) => {
+  const { page, perPage, perPageOptions } = pagination;
+  const classes = cx(
+    {
+      'content-view-pf-pagination': contentView,
```

Comment:
Why is content view excluded from view types? Can't we handle it the same way?

Reply:
i believe the `content-view-pf-pagination` class is applied in addition to the others, so it is not mutually exclusive. Is this correct @jeff-phillips-18 ?

Reply:
Correct but I'm not sure we ever want this class not added.

Reply:
made this the default `className` provided